boto3/data/cur/2017-01-06/resources-1.json

```diff
@@ -0,0 +1,44 @@
+{
+  "service": {
+    "actions": {
+      "PutReportDefinition": {
+        "request": {
+          "operation": "PutReportDefinition"
+        }
+      }
+    },
+    "hasMany": {
+      "ReportDefinitions": {
+        "request": {
+          "operation": "DescribeReportDefinitions"
+        },
+        "resource": {
+          "type": "ReportDefinition",
+          "identifiers": [
+            { "target": "ReportName", "source": "response", "path": "ReportDefinitions[].ReportName" }
+          ],
+          "path": "ReportDefinitions[]"
+        }
+      }
+    }
+  },
+  "resources": {
+    "ReportDefinition": {
+      "identifiers": [
+        { "name": "ReportName",
```

Comment:
The resource name should not be stuttered in the identifier name. I would change this from "ReportName" to "Name". This would affect a few other lines in this resource model that are targeting the identifier. Good catch on the "memberName". This is fine and should stay.